static/js/components/EmploymentTab.js

```diff
@@ -12,10 +12,14 @@ import moment from 'moment';
 
 import { generateNewWorkHistory } from '../util/util';
 import ProfileTab from "../util/ProfileTab";
-import { saveAndContinue } from '../util/profile_edit';
+import { saveProfileStep } from '../util/profile_edit';
 
 class EmploymentTab extends ProfileTab {
-  nextUrl = "/profile/privacy";
+  constructor(props) {
+    super(props);
+    this.prevUrl = "/profile/education";
+    this.nextUrl = "/profile/privacy";
```

Comment:
why not have this pattern:

```js
nextButtonText = () => (
  this.isLastTab ? "I'm done!" : "Save and Continue"
);
```

we could define that instance method on `ProfileTab` and then set instance properties on all the child classes.